### DISCUSSION TITLE
gofish is abandoned. Remove support?

### DISCUSSION THREAD

**Discussion by reegnz** [goreleaser/goreleaser#2998] [Ideas, Issue Triage, and General discussions]
Gofish has been abandoned by the project author: https://github.com/fishworks/gofish

> [](https://github.com/fishworks/gofish#this-project-is-being-archived)THIS PROJECT IS BEING ARCHIVED
>Due to the amount of time and money required to maintain this side project, I am archiving it. > Please feel free to send me an email] if you have any questions or comments.

Given how small fish (pun intended) that packaging format was, does it make sense to keep that packaging option around in goreleaser, now that it's been abandoned?

**Comment by caarlos0** [comment]
Yes, sounds good, thanks for letting me know.

Will start the deprecation process!

**Reply by caarlos0** [reply]
https://github.com/goreleaser/goreleaser/pull/2999
